web-interface/vm-consoles-modules/vm-desktop-viewer-web.adoc

```diff
@@ -0,0 +1,15 @@
+[[vm-desktop-viewer-web]]
```

Comment:
you need to add this new file to web-interface.adoc in the parent directory

Reply:
done